app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -51,13 +52,15 @@ function PasswordConfirmStep({ errors, registerField, onChange, value }: Passwor
           )}
         </Alert>
       )}
-      {errors
-        .filter(({ error }) => error instanceof PasswordSubmitError)
-        .map(({ error }) => (
-          <Alert key={error.message} type="error" className="margin-bottom-4">
-            {error.message}
-          </Alert>
-        ))}
+      {stepErrors.length > 0 && (
+        <ScrollIntoView>
+          {stepErrors.map(({ error }) => (
+            <Alert key={error.message} type="error" className="margin-bottom-4">
+              {error.message}
+            </Alert>
```

Comment:
does a second submission that gets the same error also get the scrolling behavior? or it is only the first error? I forget if a submission clears out errors or not

Reply:
> does a second submission that gets the same error also get the scrolling behavior? or it is only the first error? I forget if a submission clears out errors or not

It does, yep. I added an extra assertion to the test case in ecb793475 to check for this though, now that you raise it.